client/src/product-components/AddToCart/AddToCart.jsx

```diff
@@ -47,39 +57,53 @@ class AddToCart extends React.Component {
     });
   }
 
+  handleAddToCart() {
+    axios.post('/cart', {
+      sku_id: this.props.sizeList[this.state.selectedIndex].sku,
```

Comment:
This is a good candidate for optional chaining

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining